app/presenters/taxon_presenter.rb

```diff
@@ -36,6 +36,17 @@ def options_for_child_taxon(index:)
       track_action: (index + 1).to_s,
       track_label: child_taxons[index].preferred_url,
       track_options: {},
+      ga4_link: {
+        event_name: "navigation",
+        type: "document list",
+        index: {
+          index_link: (index + 1).to_s,
+          index_section: page_section_total.to_s,
+          index_section_count: page_section_total.to_s,
+        },
+        index_total: child_taxons.count,
+        section: I18n.t("taxons.explore_sub_topics"),
```

Comment:
I think we should add `locale: :en` here

Reply:
Good shout, have updated, thanks.